staging/src/k8s.io/kubectl/pkg/drain/filters.go

```diff
@@ -203,6 +207,9 @@ func (d *Helper) localStorageFilter(pod corev1.Pod) podDeleteStatus {
 		return makePodDeleteStatusWithError(localStorageFatal)
 	}
 
+	// TODO: this warning gets dropped by subsequent filters;
+	// consider accounting for multiple warning conditions or at least
+	// preserving the last warning message.
 	return makePodDeleteStatusWithWarning(true, localStorageWarning)
```

Comment:
The user will never see this message.  The next filter will most likely return 'ok' and the warning is swallowed.  TBD if anyone actually cares about this.

Reply:
FYI, this was not caused by this patch set.